packages/desktop-gui/src/project/project.jsx

```diff
@@ -88,6 +89,15 @@ class Project extends Component {
   _reopenProject = () => {
     projectsApi.reopenProject(this.props.project)
   }
+
+  _pingBaseUrl = () => {
+    this.props.project.clearWarning(null, true)
+    ipc.pingBaseUrl(toJS(this.props.project.resolvedConfig.baseUrl).value)
```

Comment:
This should go through `projects/projects-api.js` instead of directly calling `ipc` here.